yt/frontends/fits/fields.py

```diff
@@ -13,15 +13,52 @@
 from yt.fields.field_info_container import \
     FieldInfoContainer
 
+
 class FITSFieldInfo(FieldInfoContainer):
     known_other_fields = ()
 
     def __init__(self, ds, field_list, slice_info=None):
         super(FITSFieldInfo, self).__init__(ds, field_list, slice_info=slice_info)
         for field in ds.field_list:
-            if field[0] == "fits": 
+            if field[0] == "fits":
                 self[field].take_log = False
 
+
+class YTFITSFieldInfo(FieldInfoContainer):
+    known_other_fields = (
+        ("density", ("code_mass/code_length**3", ["density"], None)),
+        ("dark_matter_density", ("code_mass/code_length**3", ["dark_matter_density"], None)),
+        ("number_density", ("1/code_length**3", ["number_density"], None)),
+        ("pressure", ("dyne/code_length**2", ["pressure"], None)),
+        ("thermal_energy", ("erg / g", ["thermal_energy"], None)),
+        ("temperature", ("K", ["temperature"], None)),
+        ("velocity_x", ("code_length/code_time", ["velocity_x"], None)),
+        ("velocity_y", ("code_length/code_time", ["velocity_y"], None)),
+        ("velocity_z", ("code_length/code_time", ["velocity_z"], None)),
+        ("magnetic_field_x", ("gauss", [], None)),
+        ("magnetic_field_y", ("gauss", [], None)),
+        ("magnetic_field_z", ("gauss", [], None)),
+        ("metallicity", ("Zsun", ["metallicity"], None)),
+
+        # We need to have a bunch of species fields here, too
+        ("metal_density",   ("code_mass/code_length**3", ["metal_density"], None)),
+        ("hi_density",      ("code_mass/code_length**3", ["hi_density"], None)),
```

Comment:
Do you want to also add the yt standard alias names (as per [YTEP 3](https://ytep.readthedocs.io/en/master/YTEPs/YTEP-0003.html#molecular-and-atomic-species-names)) for these species fields, so for example, 'H_p0_density`?

Reply:
I talked to @jzuhone about this and he's going to address it in a different PR.